TUTORIAL.md

```diff
@@ -162,7 +162,7 @@ type CollectionMembership {
 
 To get this simplified representation, I took out all scalar fields, all field
 names, and all nullability information. What you're left with still looks kind
-of like GraphQL but lets you focus on higher level of the types and their
+of like GraphQL but lets you focus on the higher level of the types and their
```

Comment:
This still reads odd to me "higher level of the types". What about: 

> but lets you focus on the types at a higher level and their  relationships.

Reply:
Yes, I guess it is still a bit strange. To me, both the types and their relationships were focused on before, and the change is that now we are focussing on them from farther back (or higher up). In that sense perhaps it should read: 

> What you're left with still looks kind of like GraphQL, but it lets you focus on the types and their relationships from a higher level.

Reply:
Good point, I like that!

Reply:
Sounds good! I'll push an update to this PR!